components/Author/Tabs/FeedCard.tsx

```diff
@@ -198,6 +200,7 @@ function FeedCard(props: FeedCardProps) {
       )}
       data-test={isDevEnv() ? `document-${id}` : undefined}
       key={`${formattedDocType}-${id}`}
+      onClick={props.handleClick && props.handleClick}
```

Comment:
Minor but I think it might work by just specifying `props.handleClick` since otherwise it would be undefined? Could be wrong though

Reply:
I'll null chain it. I think i might have added this because something was throwing ( don't quite remember)